src/main/java/mekanism/common/integration/multipart/MultipartTileNetworkJoiner.java

```diff
@@ -0,0 +1,83 @@
+package mekanism.common.integration.multipart;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import io.netty.buffer.ByteBuf;
+import mcmultipart.api.container.IMultipartContainer;
+import mcmultipart.api.slot.EnumFaceSlot;
+import mcmultipart.api.slot.IPartSlot;
+import mcmultipart.api.world.IMultipartBlockAccess;
+import mekanism.common.base.ITileNetwork;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.IBlockAccess;
+
+public class MultipartTileNetworkJoiner implements ITileNetwork
```

Comment:
That is what I tried, initially however it causes issues when the tile entities themselves send packets instead of a capability being requested from a `TileEntity` in the world.

For example, in `mekanism.common.tile.transmitter.TileEntityLogisticalTransporter.java`:
```
@Override
protected EnumActionResult onConfigure(EntityPlayer player, int part, EnumFacing side)
{
    TransporterUtils.incrementColor(getTransmitter());
    onPartChanged(null);
    PathfinderCache.onChanged(new Coord4D(getPos(), getWorld()));
    Mekanism.packetHandler.sendToReceivers(new TileEntityMessage(new Coord4D(getPos(), getWorld()), getNetworkedData(new ArrayList())), new Range4D(new Coord4D(getPos(), getWorld())));
    player.sendMessage(new TextComponentString(EnumColor.DARK_BLUE + "[Mekanism]" + EnumColor.GREY + " " + LangUtils.localize("tooltip.configurator.toggleColor") + ": " + (getTransmitter().getColor() != null ? getTransmitter().getColor().getColoredName() : EnumColor.BLACK + LangUtils.localize("gui.none"))));

    return EnumActionResult.SUCCESS;
}
```

In this case, since there is no need to request a capability from the `TileEntity` at the world coordinates, the joiner on the server side never gets created and the ordinal headers would not be added to the packet. On the client side, when the packet is received, the packet handler would request the `TileEntity` from the world and that packet would be passed to `MultipartTileNetworkJoiner` but since the ordinal header is missing that packet would either be dropped or improperly routed.